techcity/groups/models.py

```diff
@@ -60,18 +72,28 @@ class Kind(models.TextChoices):
         blank=True,
         help_text="An optional ID to use when checking an event source",
     )
+    card_image = models.FileField(
+        upload_to=card_image_path,
+        help_text="The small image format used when displaying a card",
+    )
+    hero_image = models.FileField(
+        upload_to=hero_image_path,
+        help_text="The large image format used when displaying the group detail page",
+    )
 
     objects = GroupManager()
 
     def __str__(self):
         return self.name
 
     @property
-    def card_image(self):
+    def card_image_static(self):
+        """TODO: remove this. Using `_static` to deconflict with the field name."""
```

Comment:
This modification makes it possible to safely deploy this change without interrupting the existing content. Once everything is in place, I can remove these shims.